controllers/admin/AdminOrdersController.php

```diff
@@ -478,19 +478,63 @@ public function postProcess()
                         // Send mail to customer
                         $customer = new Customer((int)$order->id_customer);
                         $carrier = new Carrier((int)$order->id_carrier, $order->id_lang);
+                        $address = new Address((int)$order->id_address_delivery);
+
                         if (!Validate::isLoadedObject($customer)) {
                             throw new PrestaShopException('Can\'t load Customer object');
                         }
                         if (!Validate::isLoadedObject($carrier)) {
                             throw new PrestaShopException('Can\'t load Carrier object');
                         }
+                        if (!Validate::isLoadedObject($address)) {
+                            throw new PrestaShopException('Can\'t load Address object');
+                        }
+
+                        $products = $order->getCartProducts();
+
+                        $metadata = '';
+
+                        $link = Context::getContext()->link;
+
+                        foreach ($products as $product) {
+                            $metadata .= '<div itemprop="itemShipped" itemscope itemtype="http://schema.org/Product"> <meta itemprop="name" content="' . $product['product_name'] . '"/>';
```

Comment:
It's a string. So you have to insert the carriage return within the string. http://php.net/manual/en/language.types.string.php I was wrong. `\n` should be enough.

```php
$metadata .= '<div itemprop="itemShipped" itemscope itemtype="http://schema.org/Product">' "\n" . '<meta itemprop="name" content="' . $product['product_name'] . '"/>';
```